platform/index.d.ts

```diff
@@ -1,10 +1,9 @@
-// Type definitions for Platform 1.0.0
+// Type definitions for Platform 1.3
 // Project: https://github.com/bestiejs/platform.js
 // Definitions by: Jake Hickman <https://github.com/JakeH/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-
-interface PlatformStatic {
+declare interface Platform {
```

Comment:
So interface is only used as type so `declare interface` or `interface` are the same.... both of them work